src/antsibull/changelog.py

```diff
@@ -170,8 +170,12 @@ async def _get_changelog(self, version: SemVer,
         changelog_bytes = read_changelog_file(path)
         if changelog_bytes is None:
             return None
-        changelog_data = load_yaml_bytes(changelog_bytes)
-        return ChangelogData.collection(self.collection, str(version), changelog_data)
+        try:
+            changelog_data = load_yaml_bytes(changelog_bytes)
+            return ChangelogData.collection(self.collection, str(version), changelog_data)
+        except Exception as exc:  # pylint: disable=broad-except
+            print(f'WARNING: cannot load changelog of {self.collection} {version} due to {exc}')
```

Comment:
Should this using the logging apparatus instead of `print()`ing directly in a library function?

Reply:
At some point yes, I would say so, but right now other functionality in the same file also uses `print()`.

Reply:
Meh, fair enough.